### PR TITLE
Problem: there's accidental breaking change

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- revert the breaking change

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -194,6 +194,8 @@ replace (
 	// use cosmos fork of keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 	github.com/cometbft/cometbft-db => github.com/crypto-org-chain/cometbft-db v0.0.0-20230412133340-ac70df4b45f6
+	// https://github.com/cosmos/cosmos-sdk/pull/16331
```
Comment (mmsqe) [inline go.mod:197]: do we include these feature in this [diff](https://github.com/cosmos/cosmos-sdk/compare/v0.46.13-rc.0...885b47bfbe64)

Comment (mmsqe) [inline go.mod:197]: do we include these feature in this [diff](https://github.com/cosmos/cosmos-sdk/compare/v0.46.3...crypto-org-chain:cosmos-sdk:885b47bfbe64)

File: go.mod [hunk 2]
```diff
@@ -194,6 +194,8 @@ replace (
...
+	github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.46.3-0.20230530011856-885b47bfbe64
```
Comment (mmsqe) [inline go.mod:198]: do we include these feature in this [diff](https://github.com/cosmos/cosmos-sdk/compare/v0.46.13-rc.0...crypto-org-chain:cosmos-sdk:885b47bfbe64)

Comment (yihuang) [reply go.mod:198]: let's make a hotfix release

**Pull request closed without merging**